SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -45,64 +165,69 @@ public async Task StartContainersForScenarioRun(String scenarioName)
 
-            await Task.Delay(30000).ConfigureAwait(false);
-        }
+            IContainerService securityServiceTestUIContainer = SetupSecurityServiceTestUIContainer(this.SecurityServiceTestUIContainerName,
+                                                                                                   this.SecurityServiceContainerName,
+                                                                                                   this.SecurityServicePort,
+                                                                                                   testNetwork,
+                                                                                                   ("estateUIClient", "Secret1"));
 
-        public Int32 SecurityServiceTestUIPort;
-        
+            this.SecurityServiceTestUIPort = securityServiceTestUIContainer.ToHostExposedEndpoint("5004/tcp").Port;
 
-        private void SetupSecurityServiceTestUIContainer(String traceFolder)
-        {
-            // Management API Container
-            this.SecurityServiceTestUIContainer = new Builder().UseContainer().WithName(this.SecurityServiceTestUIContainerName)
-                                                         .WithEnvironment($"Authority=http://sferguson.ddns.net:55001",
-                                                                          $"ClientId=estateUIClient{this.TestId:N}",
-                                                                          "ClientSecret=Secret1")
-                                                         .UseImage("securityservicetestwebclient").ExposePort(5004)
-                                                         .UseNetwork(new List<INetworkService>
-                                                                                                                  {
-                                                                                                                      this.TestNetwork
-                                                                                                                  }.ToArray())
-                                                         .Build().Start().WaitForPort("5004/tcp", 30000);
-
-            Console.Out.WriteLine("Started Security Service");
+            Func <String, String> securityServiceBaseAddressResolver = api => $"http://127.0.0.1:{this.SecurityServicePort}";
+            HttpClient httpClient = new HttpClient();
+            this.SecurityServiceClient = new SecurityServiceClient(securityServiceBaseAddressResolver, httpClient);
+
+            this.Containers.AddRange(new List<IContainerService>
+                                     {
+                                         securityServiceContainer,
+                                         securityServiceTestUIContainer
+                                     });
+
+            //    Console.Out.WriteLine($"Security Service Test UI Port is [{this.SecurityServiceTestUIPort}]");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)